doc/related_projects.rst

```diff
@@ -232,6 +239,9 @@ Recommendation Engine packages
 - `Surprise Lib <http://surpriselib.com/>`_ Library for explicit feedback
   datasets.
 
+- `sklearn-deap <https://github.com/rsteca/sklearn-deap>`_ Use evolutionary
+   algorithms instead of gridsearch in scikit-learn.
```

Comment:
Maybe create a section "Model selection" at the end of "Interoperability and framework enhancements" and put it there, as well as the `scikit-optimize` package: they have similar goals.

Reply:
Done.